content/grants/planning_next_decade/_index.md

```diff
@@ -30,7 +30,7 @@ for the next decade of data science, we will:
 
 Our initial focus is on creating a recommendation infrastructure for
 community-wide policy and on improving common engineering infrastructure
-(see the [SPEC]({{< relref "specs" >}}) documents).
+(see the [SPEC](specs) documents).
```

Comment:
```suggestion
(see the [SPEC](/specs) documents).
```

Reply:
@jarrodmillman For my own edification, is that leading slash necessary or just a convention we're using?

Reply:
Necessary. It represents the baseURL. Without it you get this error:
```
3:06:16 AM:   ✖ FAIL load public/grants/planning_next_decade/specs
3:06:16 AM:   | operator: load
3:06:16 AM:   | expected: 200 public/grants/planning_next_decade/specs
3:06:16 AM:   |   actual: ENOENT: no such file or directory, open '/opt/build/repo/public/grants/planning_next_decade/specs'
3:06:16 AM:   |       at: public/grants/planning_next_decade/index.html:148:19 <a href="specs">...</a>
```

Reply:
See the error for 4f9bb25.